Highbar.Functions.Test/FunctionsTest.cs

```diff
@@ -26,6 +27,27 @@ public void shouldReturnTheAlwaysValue()
 
         Assert.Equal(expectedValue, actualValue);
       }
+
+      [Fact]
+      public void ShouldReturnTheAlwaysValue()
+      {
+        string expectedValue = _testString;
+        string actualValue = Always(_testString)(new object());
+
+        Assert.Equal(expectedValue, actualValue);
+      }
+    }
+
+    public class AlwaysSupply
```

Comment:
Are you bucketing the `Supplier`/`Nullary` functions together into standard `Function`s?